src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,21 @@
+.bp-ColorPickerPalette {
+    position: fixed;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    width: 146px;
```

Comment:
We should try to avoid "magic" numbers. Can we allow the content to determine the width of the parent?

Reply:
Yep we can do that instead.